packages/trpc/server/routers/loggedInViewer/teamsAndUserProfilesQuery.handler.ts

```diff
@@ -134,7 +135,7 @@ export const teamsAndUserProfilesQuery = async ({ ctx, input }: TeamsAndUserProf
       role: membership.role,
       readOnly: input?.withPermission
         ? !hasPermissionForFiltered[index]
-        : !withRoleCanCreateEntity(membership.role),
+        : !noPbacFallbackRoles.includes(membership.role),
```

Comment:
```suggestion
        : !rolesWithWriteAccess.includes(membership.role),
```

It now identifies roles with readonly access.